### PR TITLE
Sergey's fix for tests

### PR DESCRIPTION
**Pull request by therearesomewhocallmetim**

### CODE REVIEW

**Comment by therearesomewhocallmetim** [comment]
DUDE JTTP

File: storage/country_tree.hpp [hunk 1]
```diff
@@ -153,7 +153,7 @@ class CountryTree
   };
 
 private:
-  using TCountryTreeHashTable = unordered_multimap<TKey, Node *>;
+  using TCountryTreeHashTable = multimap<TKey, Node *>;
```
Comment (ygorshenin) [inline storage/country_tree.hpp]: Why not to modify instead tests? I.e. to check equality of sets, not lists?

**Comment by therearesomewhocallmetim** [comment]
"Why not modify tests instead?" - First, this modification has already been merged into the release branch. We will have conflicts if we don't merge this one (even if we then go on to modifying the tests). Second, we would have to modify three tests instead of 1 line in the code. Third, we need to ask Sergey about it :)

@syershov Please, help with the discussion

**Comment by ygorshenin** [comment]
Ah, this is already in release branch...
Okay, LGTM...

**Pull request merged**